mods/d2k/rules/aircraft.yaml

```diff
@@ -72,16 +49,17 @@ frigate:
 		Name: Frigate
 		Description: Supply spacecraft
 	Plane:
-		ROT: 5
-		Speed: 350
-		RepairBuildings: repair
+		CruiseAltitude: 1800
+		ROT: 7
+		Speed: 296	##148
```

Comment:
The "original" converted values. The ones used are multiplied to be a playable speed. I kept the comment so it's easier to go back to the original if people disagree with the amount it was increased.